Source/Scene/Cesium3DTileGroundPrimitive.js

```diff
@@ -0,0 +1,875 @@
+/*global define*/
+define([
+        '../Core/Cartesian3',
+        '../Core/Cartographic',
+        '../Core/Color',
+        '../Core/ColorGeometryInstanceAttribute',
+        '../Core/ComponentDatatype',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/destroyObject',
+        '../Core/Ellipsoid',
+        '../Core/Geometry',
+        '../Core/GeometryAttribute',
+        '../Core/GeometryAttributes',
+        '../Core/GeometryInstance',
+        '../Core/IndexDatatype',
+        '../Core/Matrix4',
+        '../Core/OrientedBoundingBox',
+        '../Core/PrimitiveType',
+        '../Core/Rectangle',
+        '../Core/TranslationRotationScale',
+        '../Renderer/Buffer',
+        '../Renderer/BufferUsage',
+        '../Renderer/DrawCommand',
+        '../Renderer/RenderState',
+        '../Renderer/ShaderProgram',
+        '../Renderer/ShaderSource',
+        '../Renderer/VertexArray',
+        '../Shaders/ShadowVolumeFS',
+        '../Shaders/ShadowVolumeVS',
+        './BlendingState',
+        './DepthFunction',
+        './Pass',
+        './StencilFunction',
+        './StencilOperation'
+    ], function(
+        Cartesian3,
+        Cartographic,
+        Color,
+        ColorGeometryInstanceAttribute,
+        ComponentDatatype,
+        defaultValue,
+        defined,
+        destroyObject,
+        Ellipsoid,
+        Geometry,
+        GeometryAttribute,
+        GeometryAttributes,
+        GeometryInstance,
+        IndexDatatype,
+        Matrix4,
+        OrientedBoundingBox,
+        PrimitiveType,
+        Rectangle,
+        TranslationRotationScale,
+        Buffer,
+        BufferUsage,
+        DrawCommand,
+        RenderState,
+        ShaderProgram,
+        ShaderSource,
+        VertexArray,
+        ShadowVolumeFS,
+        ShadowVolumeVS,
+        BlendingState,
+        DepthFunction,
+        Pass,
+        StencilFunction,
+        StencilOperation) {
+    'use strict';
+
+    function Cesium3DTileGroundPrimitive(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+
+        this._positions = options.positions;
+        this._colors = options.colors;
+        this._offsets = options.offsets;
+        this._counts = options.counts;
+        this._indexOffsets = options.indexOffsets;
+        this._indexCounts = options.indexCounts;
+        this._indices = options.indices;
+
+        this._ellispoid = defaultValue(options.ellipsoid, Ellipsoid.WGS84);
+        this._minimumHeight = options.minimumHeight;
+        this._maximumHeight = options.maximumHeight;
+        this._center = options.center;
+        this._quantizedOffset = options.quantizedOffset;
+        this._quantizedScale = options.quantizedScale;
+
+        this._boundingVolume = options.boundingVolume;
+        this._boundingVolumes = new Array(this._offsets.length);
```

Comment:
Write it to local storage, then unload, and load only when needed.  Don't actually do that, but I am open to creative ideas like that for things like this, the batch table, etc.